android/src/main/java/com/twiliovoicereactnative/NotificationUtility.java

```diff
@@ -336,6 +336,8 @@ private static String createChannelWithLowImportance(Context context) {
   private static String getDisplayName(CallInvite callInvite) {
     String title = callInvite.getFrom();
     Map<String, String> customParameters = callInvite.getCustomParameters();
+    // If "displayName" is passed as a custom parameters in the TwiML application, 
```

Comment:
```suggestion
    // If "displayName" is passed as a custom parameter in the TwiML application, 
```
Small nitpick.
Also, should we have this documented somewhere in the JS layer as well for the auto-generated docs?